sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag>Fabric.Admin_2016-05-01;Fabric.Admin_2019-05-01;Fabric.Admin_2018-10-01;</AzureApiTag>

```

Comment:
is this just 2019-05-01 or all the resource types from the above api versions?

Reply:
They are all the resource types from the above api versions, because the fabric Network/Compute still use old version.